### PR TITLE
Todo list structure 

### PR DESCRIPTION
**Pull request by fahimy143**
functionalities
add tasks
added structure for tasks
style

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+
+### Prerequisites
+
+  - IDE eg (Atom/Visual studio code)
+  - Set-up linter(HTML and CSS)
+  - webpack
+
+### Setup
+  - Clone the github repository and set up linters and webpack
+
```
Comment (TedLivist) [inline README.md:33]: - [ ] For the benefit of non-technical/unfamiliar users kindly include more specific instructions on how to clone the repo and run the app locally.

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import _ from 'lodash';
 import './style.css';
 
 const inputText = document.querySelector('input');
```
Comment (TedLivist) [inline src/index.js:3]: - [ ] There should be an array of different objects with keys `description(string)`, `completed(boolean)`, `index(integer)`.

  The objects in this array should be dynamically rendered on the HTML page using the `add` function you have in this file.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @fahimy143,

Good job so far!
✔️ Styling matches wireframe
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (fahimy143) [reply src/index.js:3]: @TedLivist I created and array I think you didn't see that

Comment (TedLivist) [reply src/index.js:3]: @fahimy143 I can see the array but it is empty. I am on this branch. See

fahimy-1

Comment (fahimy143) [reply src/index.js:3]: image
I created an object to push inside that empty array

Comment (TedLivist) [reply src/index.js:3]: @fahimy143 at this point in the project you are required to have **hardcoded** objects in this array. The values of these objects should be dynamically rendered on the HTML page.

Comment (TedLivist) [reply src/index.js:3]: @fahimy143 yes, not from the input yet. Task addition or removal will be implemented in the next project, I think.

Comment (TedLivist) [reply src/index.js:3]: @fahimy143 I can only request changes for this specific project. Or did you intend to request changes for the add-remove project on your dashboard?

You can request a second opinion if you are not satisfied with the requested changes. Here is the [link to request the S.O.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by descholar-ceo — APPROVED** [review]
Hi @fahimy143 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented the requested change ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**